modules/solid_mechanics/include/materials/CombinedPlasticityStressUpdate.h

```diff
@@ -23,8 +23,6 @@ class CombinedPlasticityStressUpdateTempl : public BackstressRadialReturnStressU
   virtual GenericReal<is_ad> computeDerivative(const GenericReal<is_ad> & effective_trial_stress,
                                                const GenericReal<is_ad> & scalar) override;
 
-  virtual void computeYieldStress(const GenericRankFourTensor<is_ad> & elasticity_tensor);
```

Comment:
My comment was wrong.  computeYieldStress should stay in public.  Its called by a noninherited class.